_posts/2020-08-05-using-open-policy-agent-with-strimzi-and-apache-kafka.md

```diff
@@ -0,0 +1,456 @@
+---
+layout: post
+title:  "Using Open Policy Agent with Strimzi and Apache Kafka"
+date: 2020-08-05
+author: jakub_scholz
+---
+
+One of the new features in Strimzi 0.19.0 is support for Kafka authorization using [Open Policy Agent](https://www.openpolicyagent.org/).
+In this blog post, we will look at this feature with more detail.
+It will explain the (dis)advantages, compare it with other supported authorization methods and look at some interesting ways how to use it.
```

Comment:
```suggestion
It will explain the advantages and disadvantages of using it, compare it with other supported authorization methods and look at some interesting ways it can be used.
```